proto/proto.go

```diff
@@ -0,0 +1,12 @@
+package proto
+
+type NodeId string
+type NodeKey uint64
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type NodeKey should have comment or be unexported